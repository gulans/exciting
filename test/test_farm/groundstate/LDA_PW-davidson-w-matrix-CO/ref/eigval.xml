<?xml version="1.0" encoding="UTF-8"?>
<eigval>
  <kpt ik="1" vkl="0.000000000000e0 0.000000000000e0 0.000000000000e0">
    <state ist="1" eigenvalue="-1.861413475450e1" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-9.798527280399e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-9.370039869116e-1" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="-3.764571501794e-1" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="-3.008273443742e-1" occupancy="2.000000000000e0"/>
    <state ist="6" eigenvalue="-3.008273443742e-1" occupancy="2.000000000000e0"/>
    <state ist="7" eigenvalue="-2.433067155057e-1" occupancy="2.000000000000e0"/>
    <state ist="8" eigenvalue="8.100392300352e-2" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="8.100392300353e-2" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="1.201134938276e-1" occupancy="0.000000000000e0"/>
  </kpt>
</eigval>
