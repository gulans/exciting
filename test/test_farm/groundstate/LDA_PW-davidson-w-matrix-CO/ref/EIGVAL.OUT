     1 : nkpt
    10 : nstsv

     1   0.000000000       0.000000000       0.000000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -18.61413475       2.000000000    
     2  -9.798527280       2.000000000    
     3 -0.9370039869       2.000000000    
     4 -0.3764571502       2.000000000    
     5 -0.3008273444       2.000000000    
     6 -0.3008273444       2.000000000    
     7 -0.2433067155       2.000000000    
     8  0.8100392300E-01   0.000000000    
     9  0.8100392300E-01   0.000000000    
    10  0.1201134938       0.000000000    

